features/environment.py

```diff
@@ -150,6 +150,16 @@ def before_feature(context, feature):
     if skip_use_case:
         feature.skip("Feature (%s) not in any use case." % feature)
 
+    try:
+        ignored_steps = context.vendor_config["ignored_steps"][feature.location.filename]
+
+        for step in ignored_steps:
+            if step == "all":
+                feature.skip("Feature (%s) requested skip by vendor." % feature)
+
+    except KeyError:
+        logging.info("ignored_steps not configured for this feature.")
```

Comment:
Is this an error? Why don't we treat "not configured" as "don't want to skip anything"?

Reply:
Do you mean not to rely on a try-except, or to have the log message instead say 'don't want to skip anything'

Reply:
(moving to slack...)